.travis.yml

```diff
@@ -51,15 +51,16 @@ install:
   - pip install dist/*.tar.gz
   - pip install h5py pillow
   - travis_wait pip install theano
-  - pip install nose hacking mock
+  - pip install pytest hacking mock
   - pip install autopep8
+  - pip install pytest-xdist
 
 script:
   - flake8
   - autopep8 -r . --global-config .pep8 --diff | tee check_autopep8
   - test ! -s check_autopep8
   - cd tests
-  - PYTHONWARNINGS='ignore::FutureWarning,module::DeprecationWarning' nosetests -a '!gpu,!slow' --with-doctest chainer_tests
+  - PYTHONWARNINGS='ignore::FutureWarning,module::DeprecationWarning' pytest -n 2 -m "not gpu and not slow" chainer_tests
```

Comment:
* How about removing `-n 2`? It's saild that multi-process testing is slower than single-process. (if it's removed, pytest-xdist dependency should be removed as well.)
* `-m` specificiation should be `not gpu and not multi_gpu and not slow`.

Reply:
Thanks. Fixed both points.